lnbits/app.py

```diff
@@ -341,10 +394,14 @@ async def listeners():
         loop.create_task(catch_everything_and_restart(invoice_listener))
         loop.create_task(catch_everything_and_restart(internal_invoice_listener))
         await register_task_listeners()
+        await register_watchdog()
+        await register_killswitch()
         # await run_deferred_async() # calle: doesn't do anyting?
 
     @app.on_event("shutdown")
     async def stop_listeners():
+        await unregister_watchdog()
```

Comment:
- `unregister_watchdog()` should not be called anymore

Reply:
commented out